### PR TITLE
Pp 1501 Updated circle.yml to latest process.

### PR DESCRIPTION
**Pull request by matthew-craig-ncino**
https://ncinodev.atlassian.net/browse/PP-1501

Updated circle.yml

- using latest deploy_toolkit
- copying resource file to artifacts directory

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -3,29 +3,46 @@
...
+    # Clone loanlifecycle/deploy_toolkit AND allow for testing of experimental branches
+    - |
+      if [ "$DEPLOY_TOOLKIT_BRANCH" == "<DEPLOY_TOOLKIT_BRANCH>" ]; then DEPLOY_TOOLKIT_BRANCH="master"; fi
+      echo "Checking out branch: $DEPLOY_TOOLKIT_BRANCH\n"
+      git clone -b $DEPLOY_TOOLKIT_BRANCH git@github.com:loanlifecycle/deploy_toolkit.git $DEPLOY_TOOLKIT
+    # Install gem bundler
+    - gem install bundler --pre
+    # Update npm binary itself (avoids red warning messages)
+    - npm install -g npm
     - sudo apt-get install --assume-yes --force-yes ruby-dev nodejs npm build-essential
```
Comment (Kevmo92) [inline circle.yml]: are sudo's necessary on line 26 and 27? Just curious.

**Review by Kevmo92 — APPROVED** [review]
Code looks good.

File: circle.yml (hunk 1 above)
Comment (Kevmo92) [reply circle.yml]: I'm testing in another repo and it does seem like sudo is needed in front of apt-get

Comment (matthew-craig-ncino) [reply circle.yml]: That would make sense these are ubuntu boxes. you need root privileges to run apt-get on system files. I don't know why you don't need sudo on bundler or npm install other than those might be considered "user" binaries, and not "system" binaries? Not sure I haven't done the homework on that one.

**Pull request closed without merging**